java/client/src/main/java/glide/api/models/ArgsBuilder.java

```diff
@@ -0,0 +1,37 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models;
+
+import com.google.protobuf.ByteString;
+import redis_request.RedisRequestOuterClass.Command.ArgsArray;
+
+// TODO docs for the god of docs
+public class ArgsBuilder {
+    ArgsArray.Builder commandArgs = null;
+
+    public ArgsBuilder() {
```

Comment:
If we wanted to make this work with the leaked pointer optimization, we should have this populate a CommandBuilder object, and conditionally either fill in ArgsValuePointer or ArgsArray. This logic is already implemented in CommandManager#populateCommandWithArgs() functions (and we would avoid intermediate conversion to ByteString for the pointer case).

Reply:
I have modified `ArgsBuilder` to any type and store it as `GlideString[]` - and finally it calls `CommandManager#populateCommandWithArgs()` (which handles the leaked ptr or not logic)

I also re-enabled the large value tests which are now passing